### PR TITLE
[17.0][MIG] account_move_budget

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by JordiMForgeFlow — APPROVED** [review]
code review

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: account_move_budget/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+In order to set up the budget items go to *Invoicing / Accounting /
+Miscellaneous / Account Move Budgets*.
```
Comment (luisDIXMIT) [inline account_move_budget/readme/USAGE.md]: The route should be Accounting / Accounting / Journals / Account Move Budgets

**Review by luisDIXMIT — COMMENTED** [review]
Looks OK but i will recomend to disallow edition on confirmed budget.

I found small typo errors.

File: account_move_budget/readme/USAGE.md (hunk 1 above)
Comment (JasminSForgeFlow) [reply account_move_budget/readme/USAGE.md]: Updated, Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**